deliver/lib/deliver/upload_metadata.rb

```diff
@@ -111,6 +111,7 @@ def upload(options)
       end
 
       v.release_on_approval = options[:automatic_release]
+      v.toggle_phased_release(enabled: !!options[:phased_release])
```

Comment:
What happens when phased releases are enabled on iTunes Connect, and you ship a new release using _fastlane deliver_ with the default value?